server/server.js

```diff
@@ -17,7 +18,11 @@ app.get('/home', (req, res) => {
     res.send(`Hello`);
 })
 
-app.post('/users/signup', async (req, res) => {
+app.post('/users/signup', check("email", "This is not a valid email").isEmail(), async (req, res) => {
+    const myValidationResult = validationResult(req).array();
+    if (myValidationResult.length > 0) {
```

Comment:
Good! This will be very handling when handling error displaying from the client, returning an array with all the errors.